### PR TITLE
Fix exports

### LINKED ISSUE #1258: things are missing in stargate/index.ts and stargate/modules/index.ts
* `createVestingAminoConverters`, `createSlashingAminoConverters` are omitted in stargate/index.ts while other amino converters are exist. (In commit `5534493` createVestingAminoConverters are add to createDefaultTypes, but createVestingAminoConverters is not added to stargate/index.ts.)
* `setupAuthzExtension`, `setupSlashingExtension` are ommited in stargate/index.ts and stargate/modules/index.ts
* `createFreegrantAminoConverters` is typo in stargate/src/modules/feegrant/aminomessages.ts

### PR DESCRIPTION
**Pull request by webmaster128**
Closes #1258

### CODE REVIEW

**Comment by loin3** [comment]
LGMT
Thank you for your quick response.

File: packages/stargate/src/index.ts [hunk 1]
```diff
@@ -25,6 +25,17 @@ export {
   AminoMsgWithdrawValidatorCommission,
   AuthExtension,
   BankExtension,
+  createAuthzAminoConverters,
+  createBankAminoConverters,
+  createDistributionAminoConverters,
+  createFeegrantAminoConverters,
+  // Alias for old typo "Freegrant" (https://github.com/cosmos/cosmjs/issues/1258)
+  createFeegrantAminoConverters as createFreegrantAminoConverters,
```
Comment (loin3) [inline packages/stargate/src/index.ts:33]: Why do you need this line? Is it because of backwards compatibility?

Comment (webmaster128) [reply packages/stargate/src/index.ts:33]: Yeah that was the idea. But then again, better remove it now.

**Comment by loin3** [comment]
LGTM, CI test is failing though.

**Pull request merged**